fiftyone/utils/cvat.py

```diff
@@ -3199,9 +3209,12 @@ def upload_samples(self, samples, backend):
             is_existing_field = label_info["existing_field"]
             only_keyframes = label_info.get("only_keyframes", False)
 
-            cvat_schema, assign_scalar_attrs = self._build_cvat_schema(
-                label_field, label_info
-            )
+            (
+                cvat_schema,
+                assign_scalar_attrs,
+                field_occ_widgets,
```

Comment:
I think a more clear name for this is `occluded_attrs`, because it does not actually contain a "widget", it just contains the names of any attributes that are tied to the occlusion widget.

This comment applies to naming throughout